drupal/recipes/wunder_next_setup/config/require_login.settings.yml

```diff
@@ -8,10 +8,14 @@ requirements:
     context_mapping:
       language: '@language.current_language_context:language_interface'
     langcodes: {  }
+  response_status:
+    id: response_status
+    negate: false
+    status_codes: {  }
   request_path:
     id: request_path
     negate: true
-    pages: "/subrequests/*\r\n/jsonapi\r\n/jsonapi/*\r\n/router/translate-path\r\n/next/preview-url\r\n/oauth/*\r\n/wunder_search/proxy\r\n/webform_rest/*\r\n/user/register\r\n/user/reset\r\n/user/reset/*\r\n/graphql\r\n/sites/default/files/sitemap.xml"
+    pages: "/subrequests/*\r\n/jsonapi\r\n/jsonapi/*\r\n/router/translate-path\r\n/next/preview-url\r\n/oauth/*\r\n/wunder_search/proxy\r\n/webform_rest/*\r\n/user/register\r\n/user/reset\r\n/user/reset/*\r\n/graphql"
```

Comment:
no more need to prevent the require_login module from triggering on the sitemap path